setup.py

```diff
@@ -49,25 +58,25 @@
 
 
 setup(
-    name="<PYPI_NAME>",
+    name="docs-build",
     use_scm_version=True,
     setup_requires=["setuptools_scm"],
-    description="""<PYPI_NAME>: <SHORT_DESCRIPTION>""",
+    description="docs-build: Build Sphinx Documentation for ApeWorX plugins",
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/<REPO_NAME>",
+    url="https://github.com/ApeWorX/docs-build",
     include_package_data=True,
-    install_requires=[],
+    install_requires=["shibuya"],
     python_requires=">=3.9,<4",
     extras_require=extras_require,
-    py_modules=["<MODULE_NAME>"],
+    py_modules=["docs_build"],
     license="Apache-2.0",
     zip_safe=False,
     keywords="ethereum",
     packages=find_packages(exclude=["tests", "tests.*"]),
-    package_data={"<MODULE_NAME>": ["py.typed"]},
+    package_data={"docs_build": ["py.typed"]},
```

Comment:
We might just want to delete this line, it is not going to work since there is no folder named `docs_build`